core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -351,15 +351,13 @@ protected void stopServing() throws Exception {
       }
     }
     if (mRPCExecutor != null) {
-      mRPCExecutor.shutdown();
+      mRPCExecutor.shutdownNow();
```

Comment:
Same question here - should we give some grace period for RPC calls to finish before interrupting them?

Reply:
This is provided to `GrpcServer` externally. After the server is done shutting down, I think this executor should be interrupted immediately as whatever remains executing probably prevented graceful server shutdown anyway.